Cache/Main.lean

```diff
@@ -44,7 +46,11 @@ $ lake exe cache get Mathlib/Algebra/Field/*.lean Mathlib/Data/*.lean
 Which will download the cache for:
 * Every Lean file inside 'Mathlib/Algebra/Field/'
 * Every Lean file inside 'Mathlib/Data/'
-* Everything that's needed for the above"
+* Everything that's needed for the above
+
+If no arguments are given, 'get', 'get!' and 'get-' download information about all files imported
+in some .lean file in the current directory or a subdirectory thereof (ignoring .lake folders).
```

Comment:
Isn't this the same as `lake exe cache get .`?

Reply:
Good question! Running this in sphere-eversion (i.e., a project downstream of mathlib) yields `uncaught exception: Unknown package directory for .` - so I would say no :-)